cargo-ledger/src/setup.rs

```diff
@@ -13,7 +13,7 @@ pub fn install_targets() {
     let sysroot_cmd = std::str::from_utf8(&sysroot_cmd).unwrap().trim();
 
     let target_files_url = Path::new(
-        "https://raw.githubusercontent.com/LedgerHQ/ledger-device-rust-sdk/cee5644d6c20ff97b13e79a30caca751b7b52ac8/ledger_device_sdk/",
+        "https://raw.githubusercontent.com/LedgerHQ/ledger-device-rust-sdk/master/ledger_device_sdk/"
```

Comment:
I'm afraid that not targeting a specific version will create incompatibilities.

If at some point with edit nanos.json, it might end up incompatible with a previous version of cargo ledger, but if cargo ledger is configured to use the last version, then this version of cargo ledger will be unusable, breaking reproducibility.

Reply:
Yes fixed 👍